oauthlib/oauth1/rfc5849/signature.py

```diff
@@ -93,37 +96,40 @@ def signature_base_string(http_method, base_str_uri,
     # 5.  The request parameters as normalized in `Section 3.4.1.3.2`_, after
     #     being encoded (`Section 3.6`).
     #
-    # .. _`Section 3.4.1.3.2`: https://tools.ietf.org/html/rfc5849#section-3.4.1.3.2
+    # .. _`Sec 3.4.1.3.2`: https://tools.ietf.org/html/rfc5849#section-3.4.1.3.2
```

Comment:
Why did you change this?

Reply:
To satisfy PEP-8 maximum line length rule.

Reply:
I'm using PyCharm IDE, so fixing the PEP-8 warnings means the file comes up with no detected issues. Otherwise, the warnings are noise that one has to ignore, and one might miss a real error/warning in amongst them.